app/javascript/components/atoms/DropdownMenuItemCheckbox.js

```diff
@@ -1,6 +1,6 @@
 import React from 'react'
 
-const dropdownMenuItemCheckbox = ({ name, item, selected }) => {
+const DropdownMenuItemCheckbox = ({ name, item, selected }) => {
```

Comment:
Not part of AC but no worries! We can keep your work on dropdown-related components, it'll give us a head start on the related story :) Kudos making use of the component inventory doc.